msg/GameState.msg

```diff
@@ -0,0 +1,67 @@
+# This message provides all information from the game controller
+# for additional information see documentation of the game controller
+# https://github.com/bhuman/GameController
+
+
+std_msgs/Header header
+
+uint8 GAMESTATE_INITAL=0
```

Comment:
Please fix this typo (see https://github.com/bit-bots/humanoid_league_msgs/issues/37)